lua/nvim-lsp-installer/server.lua

```diff
@@ -70,6 +70,23 @@ function M.Server:setup(opts)
     end
 end
 
+---@param opts table @User-defined options. This is directly passed to the lspconfig's setup() method.
+function M.Server:setup(opts)
+    self:lspconfig_setup(opts)
+    self:attach_buffers()
```

Comment:
@kylo252 Will this be problematic for your integration? The `:setup()` method will attach buffers by default now, gives a nicer public API and allows to easily deprecate the `User LspAttachBuffers` autocmd

Reply:
I was actually considering pushing this upstream since I made the changes in [`546ece0` (#1863)](https://github.com/LunarVim/LunarVim/pull/1863/commits/546ece0b58c6ab0bad24d8092f236fe7ca7b2b9b), but I made a mistake there and didn't check for for the filetype (`try_add` vs `try_add_wrapper`), which you already fixed here 😄